### PR TITLE
SEOImages: User Story 1463880, Part 20

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report:
https://contentqacats.azurewebsites.net/CheckResults/2019031700029/Report/

There are two image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

File: docs/visual-basic/programming-guide/language-features/xml/overview-of-linq-to-xml.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Visual Basic provides support for [!INCLUDE[sqltecxlinq](~/includes/sqltecxlinq-
 > [!NOTE]
 >  XML literals and XML axis properties are not supported in declarative code in an ASP.NET page. To use Visual Basic XML features, put your code in a code-behind page in your ASP.NET application.  
   
- ![link to video](../../../../visual-basic/programming-guide/language-features/xml/media/playvideo.gif "PlayVideo") For related video demonstrations, see [How Do I Get Started with LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-get-started-with-linq-to-xml) and [How Do I Create Excel Spreadsheets using LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-create-excel-spreadsheets-using-linq-to-xml).  
+ ![Icon that links to a video.](./media/overview-of-linq-to-xml/play-video-icon-example.gif) For related video demonstrations, see [How Do I Get Started with LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-get-started-with-linq-to-xml) and [How Do I Create Excel Spreadsheets using LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-create-excel-spreadsheets-using-linq-to-xml).
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/xml/overview-of-linq-to-xml.md]:
```suggestion
 ![Play button](./media/overview-of-linq-to-xml/play-video-icon.gif) For related video demonstrations, see [How Do I Get Started with LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-get-started-with-linq-to-xml) and [How Do I Create Excel Spreadsheets using LINQ to XML?](/aspnet/web-forms/videos/data-access/linq-videos-from-the-vb-team/how-do-i-create-excel-spreadsheets-using-linq-to-xml).  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. A couple of comments.

File: docs/visual-basic/programming-guide/language-features/xml/overview-of-linq-to-xml.md (hunk 1 above)
Comment (damabe) [reply docs/visual-basic/programming-guide/language-features/xml/overview-of-linq-to-xml.md]: @mairaw I've made the changes here. the file name is less than 20 characters and throws a CATS error. Do you have a preference for any additional characters/words?

Comment (mairaw) [reply docs/visual-basic/programming-guide/language-features/xml/overview-of-linq-to-xml.md]: Argh, sorry about that. Then the original name was good: play-video-icon-example.gif

**Comment by damabe** [comment]
@mairaw Fixed the file name. Updated CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019050100018/Report/

**Review by mairaw — APPROVED** [review]

**Pull request merged**